rs/ethereum/cketh/minter/src/state.rs

```diff
@@ -557,6 +557,18 @@ impl State {
     pub fn eth_balance(&self) -> &EthBalance {
         &self.eth_balance
     }
+
+    pub fn max_block_spread_for_logs_scraping(&self) -> u16 {
+        if self.evm_rpc_id.is_some() {
+            // Limit set by the EVM-RPC canister itself, see
+            // https://github.com/internet-computer-protocol/evm-rpc-canister/blob/3cce151d4c1338d83e6741afa354ccf11dff41e8/src/candid_rpc.rs#L192
+            500_u16
+        } else {
```

Comment:
The else branch will go away once everything goes through the EVM RPC canister, right?

Reply:
yes, at some point we will remove everything, but only once we know that rolling back is no longer necessary (a few months after it runs on production without any issue). That's basically step 7 in the [epic description](https://dfinity.atlassian.net/browse/XC-27).